.travis.yml

```diff
@@ -38,12 +37,14 @@ after_failure:
 matrix:
   include:
     - env: TEST="womtool validate and submodule up to date"
-      stage: lint
+      if: branch = master OR branch = develop
```

Comment:
Will this still run if we try to merge a branch into develop? Does `branch = develop` take care of that?

Reply:
Yes, this runs the check when you try to merge to develop or master